Gemfile

```diff
@@ -37,6 +37,7 @@ group :development, :test do
   gem 'rspec-rails', '~> 4.0.0.beta4'
   gem "factory_bot_rails"
   gem 'pry'
+  gem 'faker', :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'
```

Comment:
Be aware that pointing to `master` is not ideal as you could get breaking changes often.